features/step_definitions/basic_steps.rb

```diff
@@ -8,13 +8,21 @@
   visit root_path
 end
 
+Given("visit Sign up page") do
+  visit new_user_registration_path
+end
+
+
 When("I fill in {string} with {string}") do |element, value|
   fill_in element,with: value
 end
 
 When("I click {string}") do |element|
   click_on element
 end
-  
+
+Then("I am on Sign up page") do
```

Comment:
It seems same- visit sign up page and this step. Can we make it one?

Reply:
In one we are navigating to another page, in other we are just verifying url of current page.